.env

```diff
@@ -1,2 +1,2 @@
-secretKey = AwNddQ13R,
+SECRETKEY = AwNddQ13R,
```

Comment:
### Nitpick

This should not be in you version control at all, it contains very sensitive data. Add the file to your `.gitignore` list and [check this link out on how to delete it online](https://stackoverflow.com/questions/2047465/how-can-i-delete-a-file-from-git-repo/28772827)